asreview/entry_points/__init__.py

```diff
@@ -16,6 +16,7 @@
 from asreview.entry_points.auth_tool import AuthTool
 from asreview.entry_points.base import BaseEntryPoint
 from asreview.entry_points.lab import LABEntryPoint
+from asreview.entry_points.lab import WebRunModelEntryPoint
```

Comment:
I think we consider this a private entry point. Maybe we can make this explicit.